src/super_gradients/qat_from_recipe.py

```diff
@@ -13,7 +15,16 @@
 
 @hydra.main(config_path="recipes", version_base="1.2")
 def _main(cfg: DictConfig) -> None:
-    Trainer.quantize_from_config(cfg)
+    result = Trainer.quantize_from_config(cfg)
+
+    print("Validation result of quantized model:")
```

Comment:
Leftover prints ?

Reply:
Actually I left then intentionally to print the results of quantize_from_config to console.